File: 283_Move_Zeroes/283_1.c
void moveZeroes(int* nums, int numsSize){
    int i = 0, j = 0;
    for (int i = 0; i < numsSize; i++) {
        if (nums[i] != 0) {
            nums[j++] = nums[i];
        }
    }
    while (j < numsSize) {
        nums[j++] = 0;
    }
}

File: 283_Move_Zeroes/283_1.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        i = 0
        for n in nums:
            if n != 0:
                nums[i] = n
                i += 1
        while i < len(nums):
            nums[i] = 0
            i += 1
